backend/asmlink.ml

```diff
@@ -96,16 +96,17 @@ let check_cmx_consistency file_name cmxs =
 let check_consistency ~unit cmis cmxs =
   check_cmi_consistency unit.file_name cmis;
   check_cmx_consistency unit.file_name cmxs;
+  let ui_name = CU.Name.to_string (CU.name unit.name) in
```

Comment:
I'd like @lpw25 to read the diff to this file.  It's unfortunate that we have to go back to `string` here, maybe we should try to make this `CU.Name.t` now?